### PR TITLE
Updates to roadmap for 1.11 and 1.12 and new upcoming features section (WIP)

### PR DESCRIPTION
**Pull request by geojaz**
Not sure why I'm running into weird test issues when this is all docs. Sorting it out so we can get merged...

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,99 +1,46 @@
...
-### _kops 1.11_
+## UPCOMING RELEASES
 
-* Make the etcd-manager the default, deprecate the protokube-integrated approach
-* kops-server
-* Machines API support (including bare-metal)
+### kops 1.11
 
-# HISTORICAL
+* Full support for Kubernetes 1.
```
Comment (rdrgmnzs) [inline ROADMAP.md]: Is this supposed to be 1.11?

**Comment by rdrgmnzs** [comment]
One quick comment otherwise looks good.

File: ROADMAP.md [hunk 2]
```diff
@@ -1,99 +1,46 @@
...
 
-For the next few releases this means that:
 
-* 1.9.0 release target April 7th
-* 1.10 alpha.1 with release of kops 1.9.0 (April 7th)
-* 1.10 release target April 28th
-* 1.11 alpha.1 at release of kops 1.10
-* 1.11 beta.1 at release of k8s 1.11
-* 1.12 alpha.1 at release of kops 1.11 etc
+In practive, sometimes this means that kops can release months after the release of Kubernetes. We sincerely hope that is not the case, but it happens. We also realize that this is not an entirely determininstic release process, particularly with respect to networking plugins that are continually in development. We understand this is in some ways not ideal, but the community decided to continue down this path and in lieu of quickly releasing and iterating on patch versions, to release alpha and beta versions early and often. We are still working on improving this process as seen below.
```
Comment (dorongutman) [inline ROADMAP.md]: `In practive` should be `In practice`

**Comment by geojaz** [comment]
Thanks for the feedback! I think I pushed an incorrect version. Let's see how this one does...

**Comment by willthames** [comment]
@geojaz this now has some very unexpected files in it! Try a `git pull --rebase`

File: ROADMAP.md [hunk 3]
```diff
@@ -1,99 +1,51 @@
...
 
-kops 1.N.0 is released when it is believed that kubernetes 1.N.x is stable, along with all the core addons (e.g. networking).
-This can mean that kops can release months after the release of kubernetes.  It's also not a deterministic release criteria,
-particularly with some networking plugins that are supported by kops but themselves still under development.  We discussed
-this challenge in kops office hours in March 2018, and the consensus was that we want to keep this, but that we should release
-alphas & betas much earlier so that users can try out new kubernetes versions on release day.
+However, kops 1.N.x does NOT support Kubernetes 1.N+1.x. Sometimes you get lucky and kops 1.N will technically install a later version of Kubernetes, but we cannot guarantee or support this situation. As always, we recommend waiting for the official release of kops with minor version >= the version of Kubernetes you wish to install. Please see the [compatibility matrix](README.md#Compatibility_Matrix) for further questions.
```
Comment (willthames) [inline ROADMAP.md]: I still suggest wrapping lines at a reasonable line width (there seemed to be better line wrapping before this change)

Comment (geojaz) [reply ROADMAP.md]: @willthames I appreciate the suggestions. My git-fu is apparently failing me. I've ended up all turned around due to test flakes yesterday and now I'm in a weird place :) rebasing and squashing and we'll see if we can get this back on track!

As for the line wraps- I've generally let the browser/github deal with that. Is this paragraph displaying in a strange way for you? It looks reasonable to me- but if you can help me improve it, please do. I'm open to suggestions, I just don't know what you're looking for exactly. Maybe a screenshot would help!
Thanks again

Comment (willthames) [reply ROADMAP.md]: @geojaz - shorter lines tend to (but don't always) make for smaller diffs and simpler reviews (the fewer words on a line, the easier it is to say which line has the typo).

It's also slightly easier to get to a word in a text editor when using the keyboard as you can get to the right line first and then the right word.

It's displaying fine in github - I've definitely come across times where that's not the case (I seem to recall bitbucket doesn't do so well, but that's not an issue for this repo)

Comment (geojaz) [reply ROADMAP.md]: @willthames Cool, I'm picking up what your putting down... understood. i'll come back to this tomorrow with fresh eyes and see what I can do for ya.

**Comment by justinsb** [comment]
/lgtm
/retest

Thanks @geojaz  .... we might have to tweak some of the 1.11 goals (thinking about etcd3) but this is much better

**Pull request merged**